src/wp-includes/block-patterns.php

```diff
@@ -196,9 +218,10 @@ function _load_remote_block_patterns( $deprecated = null ) {
 		}
 		$patterns = $response->get_data();
 
-		foreach ( $patterns as $settings ) {
-			$pattern_name = 'core/' . sanitize_title( $settings['title'] );
-			register_block_pattern( $pattern_name, (array) $settings );
+		foreach ( $patterns as $pattern ) {
+			$normalized_pattern = _normalize_remote_pattern( $pattern );
+			$pattern_name       = 'core/' . sanitize_title( $normalized_pattern['title'] );
+			register_block_pattern( $pattern_name, (array) $normalized_pattern );
```

Comment:
```suggestion
			register_block_pattern( $pattern_name, $normalized_pattern );
```

The type cast is not needed because `_normalize_remote_pattern()` will always return an array data type.